extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -128,26 +133,49 @@ public function getContentRelatedTo($title, $universeName=null, $limit=5, $ignor
 	 */
 	private function relatedContentApi() {
 		/** @var ApiProvider $apiProvider */
-		$apiProvider = Injector::getInjector()->get(ApiProvider::class);
+		$apiProvider = Injector::getInjector()->get( ApiProvider::class );
 		/** @var RelatedContentApi $api */
-		$api = $apiProvider->getApi(self::SERVICE_NAME, RelatedContentApi::class);
-		$api->getApiClient()->getConfig()->setCurlTimeout(self::TIMEOUT);
+		$api = $apiProvider->getApi( self::SERVICE_NAME, RelatedContentApi::class );
+		$api->getApiClient()->getConfig()->setCurlTimeout( self::TIMEOUT );
 
 		return $api;
 	}
 
-	private function formatTitle(Content $content) {
+	private function formatTitle( Content $content ) {
 		global $wgContLang;
 
-		if ($content->getContentType() == "Discussion Thread") {
+		if ( $content->getContentType() == "Discussion Thread" ) {
 			return $wgContLang->truncate(
-					"[Discussions] {$content->getTitle()}",
-					self::DISCUSSION_THREAD_TITLE_MAX_LENGTH);
+					$content->getTitle(),
+					self::DISCUSSION_THREAD_TITLE_MAX_LENGTH );
 		}
 
 		return $content->getTitle();
 	}
 
+	private function getAuthor( $metadata ) {
+		if ( array_key_exists( "authorName", $metadata ) ) {
+			return $metadata["authorName"];
+		} else {
+			return "";
+		}
+
+	}
+
+	private function getRecirculationContentType( $contentType ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)